code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -40,6 +40,59 @@
 	icon_state = "fake-moustache"
 	flags_inv = HIDEFACE
 
+/obj/item/clothing/mask/fakemoustache/italian
+	name = "italian moustache"
+	desc = "Made from authentic Italian moustache hairs. Gives the wearer an irresistable urge to gesticulate wildly."
+	icon_state = "fake-moustache"
+	flags_inv = HIDEFACE
+
+/obj/item/clothing/mask/fakemoustache/italian/speechModification(message)
+	if(copytext(message, 1, 2) != "*")
+		message = " [message] "
+		message = replacetext(message," mom "," mamma ")
+		message = replacetext(message," dad "," pappa ")
+		message = replacetext(message," baby ",pick(" bambino "," little sausage roll "))
+		message = replacetext(message,"spicy","a-spicy")
+		message = replacetext(message," friend "," enemy ")
+		message = replacetext(message," enemy "," friend ")
+		message = replacetext(message,"traitor","mafioso")
+		message = replacetext(message," operative "," greek ")
+		message = replacetext(message," op "," greek ")
+		message = replacetext(message," ops "," greeks")
+		message = replacetext(message," operative "," greek")
+		message = replacetext(message," operatives "," greeks")
+		message = replacetext(message," nuke"," spiciest-a meatball")
+		message = replacetext(message," good"," molto bene")
+		message = replacetext(message," why"," for-a what reason")
+		message = replacetext(message," bad"," molto male")
+		message = replacetext(message," sing "," sing-a ")
+		message = replacetext(message," cook "," cook-a ")
+		message = replacetext(message," want "," want-a ")
+		message = replacetext(message," what's "," what's-a ")
+		message = replacetext(message," shitcurity"," carabinieri")
+		message = replacetext(message," shitsec"," carabinieri")
+		message = replacetext(message," assistant "," goombah ")
+		message = replacetext(message," greytide "," curvisti ")
+		message = replacetext(message," greytider "," curvisti ")
+		message = replacetext(message," captain "," capitano ")
+		message = replacetext(message," sec "," polizia ")
+		message = replacetext(message," security "," polizia ")
+		message = replacetext(message,"cheese",pick("parmesano","gorgonzola"))
+		message = replacetext(message," meat",pick(" pepperoni"," prosciutto"))
+		message = replacetext(message,"hello",pick("ciao","buongiorno"))
+		message = replacetext(message," bye ",pick(" ciao "," arrivederci "))
+		message = replacetext(message,"thanks","grazie")
+		message = replacetext(message," wine"," vino")
+		message = replacetext(message,"spaghetti", "SPAGHETT")
+		for(var/i in 1 to length(message))//append -a to about 50% of the words
+			if(message[i] == " " && i > 1 && prob(50))
+				if((65 <= text2ascii(message[i-1]) <= 90) || (97 <= text2ascii(message[i-1]) <= 122))//if the previous character is a letter, upper or lowercase
+					replacetext(message[i], " ", "-a ")

```

Comment:
apparently there's a better way of doing this via regex that im currently struggling to wrap my head around

Reply:
also this doesn't function and im gay